packages/api/amplify_api/example/integration_test/graphql_tests.dart

```diff
@@ -561,7 +561,7 @@ void main() {
         // With stream established, exec callback with stream events.
         final subscription = await _getEstablishedSubscriptionOperation<T>(
             subscriptionRequest, (event) {
-          if (event.errors.isNotEmpty) {
+          if (event.hasErrors) {
             fail('subscription errors: ${event.errors}');
           }
           dataCompleter.complete(event);
```

Comment:
request: Could we comment out or have an early return in the group below about subscriptions along with a TODO comment to re-enable them once subscriptions implemented. I would prefer to have failing integ test suites and I think all these should pass here (except subscriptions), which is awesome!